app/test/prepare_proposal_test.go

```diff
@@ -184,9 +183,23 @@ func TestPrepareProposalFiltering(t *testing.T) {
 		blobfactory.NestedBlobs(
 			t,
 			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 3),
-			[][]int{{100}, {1000}, {420}},
```

Comment:
[no change needed] out of curiosity why were these 3 blob txs pruned before? They are all smaller than the max tx size.

Reply:
because they were running out of gas